source/_docs/composer.md

```diff
@@ -1,175 +1,87 @@
-
- ```json
- "require": {
-   "composer/installers": "^1.0.21",
-   "koodimonni/composer-dropin-installer": "*",
-   "wpackagist-plugin/wp-cfm": "1.*",
-   "wpackagist-plugin/wp-redis": "0.4.0"
-   },
-   "extra": {
-     "installer-paths": {
-       "wp-content/plugins/{$name}/": ["type:wordpress-plugin"],
-       "wp-content/mu-plugins/{$name}/": ["type:wordpress-muplugin"],
-       "wp-content/themes/{$name}/": ["type:wordpress-theme"]
-       },
-     "dropin-paths": {
-        "wp-content": [
-        "package:wpackagist-plugin/wp-redis:object-cache.php"
-      ]
-    }
-  }
- ```
+Composer is used to fetch dependencies declared by the project as part of a CircleCI build step. This ensures that the final composed build results are installed on Pantheon:
+
+![Artifact Deployment](/source/docs/assets/images/artifact-deployment.png)
+
+### GitHub Pull Requests
+One advantage of managing code this way is that it keeps the change sets (differences) for pull requests as small as possible. If a pull request upgrades several dependencies, only the dependency metadata file will change; the actual code changes in the upgraded dependencies themselves are not shown.
+
+GitHub pull requests (PRs) are a formalized way of reviewing and merging a proposed set of changes to the source repository. When one member of a development team makes changes to a project, all of the files modified to produce the feature are committed to a separate branch, and that branch becomes the basis for the pull request. GitHub allows other team members to review all of the differences between the new files and their original versions, before merging the PR to accept changes.
+
+In this workflow, a [Multidev](/docs/multidev/) environment is created on Pantheon for each pull request branch on GitHub. Work in these environments can also be committed back to the same branch for review on GitHub. When a pull request is merged into the default branch on GitHub, the result is deployed to the Dev environment on Pantheon:
+
+![Multidev PR workflow](/source/docs/assets/images/pr-workflow/github-pr-diagram.png)
+
+### Automated Tests
+
+It is also common to set up automated tests to confirm that the project is working as expected; when tests are available, GitHub will run them and display the results of the tests with the pull request. Working on projects with comprehensive tests increases the development team's confidence that submitted pull requests will work correctly when they are integrated into the main build.
+
+### Scaling Considerations
+We recommend the "source" and "artifact" workflow for single site use cases, and for most use cases involving larger site portfolios such as EDUs. You can create a scaffold repository based off our example repositories and customize it to your liking, then use the scaffold to create new sites similar to how you would create the codebase for a [Custom Upstream](/docs/custom-upstream/).
```

Comment:
I'm not 100% on the term "scaffold" or what a good alternative is.. 

Also, would love a spot check on technical content / advice. Does this work? Is it scalable as described? Do we need to provide a separate doc with explicit steps on how to do this?

Reply:
Other options:
Base
Core

Reply:
I usually call these "template" repositories.